Snakefile

```diff
@@ -13,7 +13,7 @@ if not exists("config.yaml"):
 
 configfile: "config.yaml"
 
-COSTS="data/costs.csv"
+COSTS="resources/costs.csv"
```

Comment:
Maybe:

if config["enable"].get("retrieve_cost_data", True):
    COSTS = "resources/costs.csv"
else:
    COSTS = "data/costs.csv"

This would allow using automatically the `data/costs.csv` if `retrieve_cost_data: False`